sqlalchemy-stubs/sql/elements.pyi

```diff
@@ -24,7 +24,7 @@ def collate(expression, collation): ...
 def between(expr, lower_bound, upper_bound, symmetric: bool = ...): ...
 def literal(value, type_: Optional[Any] = ...): ...
 def outparam(key, type_: Optional[Any] = ...): ...
-def not_(clause): ...
+def not_(clause: ClauseElement) -> ClauseElement: ...
```

Comment:
i guess the exact same can be done for the `any_()` and `all_()` functions.

https://docs.sqlalchemy.org/en/13/core/sqlelement.html

Reply:
Maybe, but I wasn't sure, they are defined in a different way, which made it more difficult for me to tell exactly what types they use: https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/sql/expression.py#L178